python/GafferUI/Viewer.py

```diff
@@ -340,7 +340,9 @@ def __init__( self, view ) :
 			self.primaryToolChangedSignal = Gaffer.Signal0()
 
 			if len( self.tools ) :
-				self.tools[0]["active"].setValue( True )
+				autoActivate = Gaffer.Metadata.value( self.tools[0], "viewer:shouldAutoActivate" )
+				if autoActivate is None or autoActivate :
+					self.tools[0]["active"].setValue( True )
```

Comment:
I feel like there must be a better solution to this, but I can't put my finger on it. Let's at least not advertise the new metadata in the release notes...